File: Plugin/BlockelotPlugin/src/main/java/com/Blockelot/worldeditor/http/BlockBankInventoryResponse.java
package com.Blockelot.worldeditor.http;

/**
 *
 * @author geev
 */
public class BlockBankInventoryResponse {

    private String Uuid;
    private String Auth;

    private BlockBankInventoryItem[] Items;
    private Boolean Success;

    public Boolean getSuccess() {
        return Success;
    }

    public void setSuccess(Boolean f) {
        Success = f;
    }

    public BlockBankInventoryItem[] getItems() {
        return Items;
    }

    public void setItems(BlockBankInventoryItem[] itms) {
        Items = itms;
    }

    public String getUuid() {
        return this.Uuid;
    }

    public void setUuid(String uuid) {
        this.Uuid = uuid;
    }

    public void setAuth(String auth) {
        Auth = auth;
    }

    public String getAuth() {
        return Auth;
    }

}

File: BackendWebsite/Minecraft Site/Source/ServerSite/DataContracts/BlockBank/BlockBankInventoryResponse.cs
namespace ServerSite.DataContracts.BlockBank
{
    public class BlockBankInventoryResponse
    {
        public string Uuid { get; set; }
        public string Auth { get; set; }
        public bool Success { get; set; }
        public BlockBankInventoryItem[] Items { get; set; }
    }
}